src/app/page.tsx

```diff
@@ -0,0 +1,100 @@
+import Link from 'next/link';
+import Footer from '@/components/footer';
+
+export default function Home() {
+  return (
+    <>
+      <section className="min-h-[94vh] w-full py-12 lg:py-14 xl:py-24 2xl:py-28">
+        <div className="container mx-auto space-y-8 px-4 md:px-6">
+          <div className="flex flex-col items-center space-y-4 text-center">
+            <div className="space-y-4">
+              <h1 className="text-3xl font-bold tracking-tighter sm:text-4xl md:text-5xl lg:text-6xl/none">
+                JobSync by tapaScript Community
+              </h1>
+            </div>
+            <p className="mx-auto max-w-[700px] text-gray-700 md:text-lg">
+              Edit <span className="font-bold">app/page.tsx</span> to start
+            </p>
+          </div>
+          <div className="grid grid-cols-1 gap-8 sm:grid-cols-2 lg:grid-cols-4">
+            {TECH_STACK.map((item, id) => (
+              <Link key={id} href={item.link} target="_blank">
+                <div className="transform rounded-lg bg-white p-6 shadow-md transition-transform hover:scale-105 sm:h-[192px] md:h-[164px] lg:h-[219px] xl:h-[179px] 2xl:h-[163px]">
+                  <h3 className="mb-2 text-xl font-bold">{item.name}</h3>
+                  <p className="text-gray-500">{item.description}</p>
+                </div>
+              </Link>
+            ))}
+          </div>
+        </div>
+      </section>
+      <Footer />
+    </>
+  );
+}
+
+const TECH_STACK = [
```

Comment:
can this be taken to a different file like `src/config/site.ts`

Reply:
We wil remove all the code in the page file in next PR, so not touching anything that came as part of template.